terraform/azure/examiner.tf.j2

```diff
@@ -62,7 +62,12 @@ resource "azurerm_virtual_machine" "examiner_vm" {
     }
 
     os_profile_linux_config {
-        disable_password_authentication = false
+        disable_password_authentication = true
+
+        ssh_keys {
+            path     = "{{ examiner.authorized_keys_file }}"
```

Comment:
I don't really understand how this rendering works, but as far as I see in the `deployment.azure.yaml.example` the `authorized_keys_files` and `public_key_file` are in `common` entry, not in `examiner`.

The same for `iscsi` and `qdevice`. I guess all of them should import these variables from `common`, I don't think any user will specify different keys for each of the machines.

Reply:
It will be okay.

Reply:
I have looked a bit into the code, I think @aleksei-burlakov  is right about this.

This is because  there is no `"{{ common.authorized_keys_file }}"`, and all yaml vars are inherited from common_var but then distributed to the other namespaces even if same.

Although it is something we might need to consider for the future to improve it is possible to have such mechanism.